Healthy/Classes/Extensions/UIButton+Style.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  UIButton+Style.swift
+//  Healthy
+//
+//  Created by Ahmed Nasr on 13/05/2023.
+//
+
+import UIKit
+
+
+//MARK: - button style -
+//
+extension UIButton {
+    enum buttonStyle {
+        case primaryButton
```

Comment:
Since we are already inside an enum named `ButtonStyle`, we don't have to add `button` prefix for before each case 
```suggestion
        case primary
```